src/DynamoRevit/DynamoRevitApp.cs

```diff
@@ -283,14 +284,16 @@ private void UnsubscribeApplicationEvents()
             proxy = null;
         }
 
-        private void SubscribeAssemblyResolvingEvent()
+        private void SubscribeAssemblyEvents()
         {
             AppDomain.CurrentDomain.AssemblyResolve += ResolveAssembly;
         }
 
-        private void UnsubscribeAssemblyResolvingEvent()
+      
+        private void UnsubscribeAssemblyEvents()
         {
             AppDomain.CurrentDomain.AssemblyResolve -= ResolveAssembly;
+            //AppDomain.CurrentDomain.AssemblyLoad -= AssemblyLoad;
```

Comment:
Remove comment? It looks like this file could be left unchanged now but I don't think the renaming of the methods hurts...